packages/steveo/src/consumers/sqs.ts

```diff
@@ -42,11 +42,16 @@ class SqsRunner extends BaseRunner implements IRunner {
     return bluebird.map(
       messages,
       async message => {
+        this.logger.info(message, `Message received for task: ${topic}`);
         const params = JSON.parse(message.Body as string);
         await this.wrap({ topic, payload: params }, async c => {
           this.logger.info(message, `Message received for task: ${c.topic}`);
           let resource: Resource | null = null;
-          const runnerContext = getContext(c.payload);
+          const { _meta: messageContext, ...data } = c.payload;
+          const runnerContext = {
+            ...getContext(c.payload),
+            ...messageContext,
+          };
```

Comment:
This logic should sit in the getContext method. 
and it should return 
```
  return {
    duration,
    ..meta,
  };
```

like in the chat we had https://github.com/ordermentum/steveo/compare/main...from-chat#diff-d6f1d1be5af1f83150e3506c5a889a370794c0835fbfcbcb87c527a0c22e487e

Reply:
Ok. I wasn't sure the reason getContext always returned a new object.
I will change it.

Reply:
Agree, I think we have also came across this during our catch up session